docs/conf.py

```diff
@@ -52,6 +58,12 @@
 # Customize display of autosummary entries
 autosummary_imported_members = True
 
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3", None),
+    "pandas": ("https://pandas.pydata.org/pandas-docs/stable/", None),
+    "numpy": ("https://numpy.org/doc/stable/", None),  # optional but handy
```

Comment:
Not sure if we have any links, but adding pyarrow here might be good

Reply:
Great idea!  Will do.

Reply:
Unfortunately, I had to revert the change, since pyarrow doesn't have an intersphinx inventory like the other packages.